### PR TITLE
Add script to login to AWS using aws-sso-util

### PR DESCRIPTION
**Pull request by doktor500**
## Description

Add script to login to AWS using [aws-sso-util](https://github.com/benkehoe/aws-sso-util)

## Type of change

- [X] New script command

## Dependencies / Requirements

It requires to have [aws-sso-util](https://github.com/benkehoe/aws-sso-util) installed and configured

## Checklist

- [X] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/developer-utils/aws/aws-sso-util.sh [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# Install via pipx: `pipx install aws-sso-util` following the instructions on https://github.com/benkehoe/aws-sso-util
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title AWS SSO
+# @raycast.mode silent
+
+# Optional parameters:
+# @raycast.icon images/aws-sso-util.png
+# @raycast.packageName Developer Utilities
```
Comment (unnamedd) [inline commands/developer-utils/aws/aws-sso-util.sh:13]:
```suggestion
# @raycast.packageName AWS
```

File: commands/developer-utils/aws/aws-sso-util.sh [hunk 2]
```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Dependency: This script requires `aws-sso-util` installed: https://github.com/benkehoe/aws-sso-util
+# Install via pipx: `pipx install aws-sso-util` following the instructions on https://github.com/benkehoe/aws-sso-util
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title AWS SSO
```
Comment (unnamedd) [inline commands/developer-utils/aws/aws-sso-util.sh:8]: I'd prefer to use `Single Sign-On` here, it's more complete and descriptive, but of course, you are more aware than me how people used to refer to it on AWS world. So, feel free to accept this suggestion or not, it's not mandatory.

```suggestion
# @raycast.title Single Sign-On
```

**Review by unnamedd — CHANGES_REQUESTED** [review]

**Review by unnamedd — APPROVED** [review]
Thanks for the change @doktor500! 

It looks good to me!

**Comment by doktor500** [comment]
Thanks, are we happy merging it then?

**Comment by unnamedd** [comment]
@doktor500 it will be merged as soon as possible! The Raycast Team is on holiday! ;)

**Comment by dehesa** [comment]
Hey @doktor500, sorry for the delay, but as @unnamedd said most of the team is on holidays. In any case, I will merge it right away. Thank you for the contribution.

**Comment by doktor500** [comment]
Thanks!

**Pull request merged**